.github/workflows/ci.yaml

```diff
@@ -0,0 +1,141 @@
+name: ci
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+
+  test_node:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        node: [ '14', '17' ]
+        os: [ windows-latest, ubuntu-latest ]
+    name: Test with Node v${{ matrix.node }} on ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node }}
+    - uses: bahmutov/npm-install@v1
+    - run: |
+        yarn build
+        yarn test
```

Comment:
I'm not using yarn on this repo, does this command refer to my package.json's build and test commands? If it's the case, could we replace this by `npm run ...`?

Reply:
Sure thing, it's done! 
I though you where using yarn since you have a `yarn.lock` file that is more recent that your `package-lock.json`.

Reply:
Ah right! I use yarn at Datadog, I must have mistakenly used `yarn add` instead of `npm i` once. I'll remove the lock file then. Thanks for updating